git-filter-branch.sh

```diff
@@ -83,15 +83,21 @@ set_ident () {
 	finish_ident COMMITTER
 }
 
-if test -z "$FILTER_BRANCH_SQUELCH_WARNING$GIT_TEST_DISALLOW_ABBREVIATED_OPTIONS"
+if test -z "$FILTER_BRANCH_SQUELCH_WARNING$GIT_TEST_DISALLOW_ABBREVIATED_OPTIONS" && \
```

Comment:
The backslash is actually not needed, `&&` already implies a continuation line.

Reply:
Fixed in latest commit bef1de4.